packages/dom/src/dom/is-form-element.js

```diff
@@ -4,17 +4,28 @@
 import isInputOrTextArea from './is-input-or-text-area';
 
 /**
- *
  * Detects if element is a form element.
  *
- * @param {Element} element The element to check.
+ * @param {Element}                             element The element to check.
+ * @param {{ includeContentEditable: boolean }} options Options object.
  *
  * @return {boolean} True if form element and false otherwise.
  */
-export default function isFormElement( element ) {
+export default function isFormElement(
+	element,
+	options = { includeContentEditable: false }
+) {
 	const { tagName } = element;
 	const checkForInputTextarea = isInputOrTextArea( element );
+	const checkContentEditable =
```

Comment:
The default value of checking `contenteditable="true"` is false for this function but I needed this extended to handle tabbing in blocks such as the table block.

Reply:
Do we need this option? Can't we just make it the default? This function was added to check form elements in placeholders. Not sure why it was ever moved to the dom package as an API.

I think we're also missing links here.

Why can't we just use check the tabbable API (findNext) and skip over blocks?

Reply:
@ellatrix I moved it because this was being used in multiple places.

https://github.com/WordPress/gutenberg/pull/39461

I will try to remove this and see if E2E catches bugs.

Reply:
Oh yes, I do not think we will be able to do this. Seems like this backfires because focus.tabbable will always have somewhere to go since blocks have tabindex of 0. Need to ensure it is a form field or something we want to move to. Unless you see another way around this...